packages/react-composites/tests/browser/common/server.ts

```diff
@@ -39,4 +39,11 @@ const stopServer = (): Promise<void> =>
     server.close(() => {
       resolve();
     });
+
+    // closeAllConnections() is only available from Node v18.02
+    if (server.closeAllConnections) {
+      server.closeAllConnections();
+    } else {
+      setTimeout(() => resolve(), 5000);
```

Comment:
sorry what is this 5000 timeout for?

Reply:
just leaving it some time in case server.close callback eventually gets called to resolve the promise